app/index.html

```diff
@@ -1,11 +1,13 @@
 <!DOCTYPE html>
 
 <head>
+	<base href='/' />
```

Comment:
All paths should be relative as we don't know what the hosting environment will look like, and it may be hosted on a subpath.  On the same note, we can't setup a `<base href='/' />` because the file server may be serving on a subpath.

 (same comment for other paths in this PR)